<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fenster</class>
 <widget class="QMainWindow" name="Fenster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fenster</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBoxEingabe">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Eingabe</string>
    </property>
    <widget class="QLabel" name="labelCe">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Obere Grenze:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPlusCe">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelPlusCa">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCeIm">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCaIm">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelCa">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Untere Grenze:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCaRe">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCeRe">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelMalDimension">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDimensionHoehe">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelDimension">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bildgröße:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDimensionBreite">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelICa">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>i</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelICe">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>i</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBerechnen">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Berechnen</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditIter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelIter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>149</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Iterationen:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRuecksetzen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Rücksetzen</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <widget class="QLabel" name="labelStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Bereit</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStatusIter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Iterationen: </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>251</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Navigation</string>
    </property>
    <widget class="QPushButton" name="pushButtonHoch">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>↑</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonZentrieren">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>⌂</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRunter">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>↓</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLinks">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>←</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRechts">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>→</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonZoomEin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonZoomAus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionExport_als_png"/>
    <addaction name="actionWerte_speichern"/>
    <addaction name="actionWerte_laden"/>
    <addaction name="separator"/>
    <addaction name="actionSchliessen"/>
   </widget>
   <widget class="QMenu" name="menuIterationen">
    <property name="title">
     <string>Iterationen</string>
    </property>
    <addaction name="action20"/>
    <addaction name="action50"/>
    <addaction name="action80"/>
    <addaction name="action100"/>
    <addaction name="action120"/>
    <addaction name="action150"/>
    <addaction name="action200"/>
    <addaction name="action400"/>
    <addaction name="action600"/>
    <addaction name="action800"/>
    <addaction name="action1000"/>
    <addaction name="action1500"/>
    <addaction name="action2000"/>
   </widget>
   <widget class="QMenu" name="menuFarbe">
    <property name="title">
     <string>Farbe</string>
    </property>
    <addaction name="actionStandard"/>
    <addaction name="separator"/>
    <addaction name="actionRot"/>
    <addaction name="actionGr_n"/>
    <addaction name="actionGelb"/>
    <addaction name="actionBlau"/>
    <addaction name="actionOrange"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuIterationen"/>
   <addaction name="menuFarbe"/>
  </widget>
  <action name="actionExport_als_png">
   <property name="text">
    <string>Export als .png ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionWerte_speichern">
   <property name="text">
    <string>Werte speichern</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSchliessen">
   <property name="text">
    <string>Schließen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action20">
   <property name="text">
    <string>20</string>
   </property>
  </action>
  <action name="action50">
   <property name="text">
    <string>50</string>
   </property>
  </action>
  <action name="action80">
   <property name="text">
    <string>80</string>
   </property>
  </action>
  <action name="action100">
   <property name="text">
    <string>100</string>
   </property>
  </action>
  <action name="action120">
   <property name="text">
    <string>120</string>
   </property>
  </action>
  <action name="action150">
   <property name="text">
    <string>150</string>
   </property>
  </action>
  <action name="action200">
   <property name="text">
    <string>200</string>
   </property>
  </action>
  <action name="action400">
   <property name="text">
    <string>400</string>
   </property>
  </action>
  <action name="action600">
   <property name="text">
    <string>600</string>
   </property>
  </action>
  <action name="action800">
   <property name="text">
    <string>800</string>
   </property>
  </action>
  <action name="action1000">
   <property name="text">
    <string>1000</string>
   </property>
  </action>
  <action name="action1500">
   <property name="text">
    <string>1500</string>
   </property>
  </action>
  <action name="action2000">
   <property name="text">
    <string>2000</string>
   </property>
  </action>
  <action name="actionStandard">
   <property name="text">
    <string>Standard</string>
   </property>
  </action>
  <action name="actionRot">
   <property name="text">
    <string>Rot</string>
   </property>
  </action>
  <action name="actionGr_n">
   <property name="text">
    <string>Grün</string>
   </property>
  </action>
  <action name="actionGelb">
   <property name="text">
    <string>Gelb</string>
   </property>
  </action>
  <action name="actionBlau">
   <property name="text">
    <string>Blau</string>
   </property>
  </action>
  <action name="actionOrange">
   <property name="text">
    <string>Orange</string>
   </property>
  </action>
  <action name="actionViolett">
   <property name="text">
    <string>Violett</string>
   </property>
  </action>
  <action name="actionWerte_laden">
   <property name="text">
    <string>Werte laden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
